<<term=True>>=
print(123)
@
<<term=True>>=
a = 42
print(a)
@
Test output.
<<term=True>>=
2 + 2 * 2 ** 2
@
<<term=True>>=
for i in range(3):
    print(i)
@
<<term=True>>=
for i in range(3):
    print(i)
    print(i**2)
@
This code does not work as expected
\begin{verbatim}
for i in range(3):
    print(i)

    print(i**2)
\end{verbatim}


\begin{verbatim}
>>> for i in range(3):
...     print(i)
...
...     print(i**2)
...
0
0
1
1
2
4

\end{verbatim}

<<term=True>>=
for i in range(3):
    print(i)

print(1 + 2)
@
<<term=True>>=
for i in range(2):
    print(i)
    if i > 0:
        print("one")
@
<<term=True>>=
for i in range(3):
    print(i)
    if i == 1:
        raise Exception("foo")

print(42)
@
<<term=True,name='SyntaxError'>>=
def f
print(42)
@
<<term=True,name='IndentationError'>>=
def f():
    print(12)
  print(8)
print(42)
@
Empty code chunk does not work

\begin{verbatim}
>>>

\end{verbatim}

End
<<term=True>>=
@
